yapapi/config.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass, field
+from functools import partial
+import os
+from typing import Optional
+
+
+class MissingConfiguration(Exception):
+    def __init__(self, key: str, description: str):
+        self._key = key
+        self._description = description
+
+    def __str__(self):
+        return f"Missing configuration for {self._description}. Please set env var {self._key}."
+
+
+@dataclass
+class ApiConfig:
+    """
+    Yagna low level API configuration
+    Attributes:
+        subnet_tag: use only providers in the subnet with the subnet_tag name.
+            Uses `YAGNA_SUBNET` environment variable, defaults to `None`
+        payment_driver: name of the payment driver to use.
+            Uses `YAGNA_PAYMENT_DRIVER` environment variable, defaults to `erc20`.
+            Only payment platforms with the specified driver will be used
+        payment_network: name of the network to use.
+            Uses `YAGNA_NETWORK` environment variable, defaults to `rinkeby`.
```

Comment:
you lost `_PAYMENT` here... but still see my other comment -> I strongly feel this parameter should not be included in `ApiConfig`